lib/dogapi/common.rb

```diff
@@ -224,4 +224,14 @@ def Dogapi.validate_tags(tags)
       raise ArgumentError, "Each tag needs to be a string. Current value: #{tag}" unless tag.is_a? String
     end
   end
+
+  # Very simplified hash with indifferent access - access to string or symbol
+  # keys via symbols. E.g.:
+  # my_hash = { 'foo' => 1 }
+  # Dogapi.symbolized_access(my_hash)
+  # my_hash[:foo] # => 1
+  def Dogapi.symbolized_access(hash)
```

Comment:
What's the reasoning behind this one ?
We already have a few endpoints that use a hash for options, but aren't using this symbolized access.

Reply:
Looking across the other usages, all of the other `options` are merged into the `body` without modification. In a few cases here we construct a deeper object in the body based on the flat options passed in.